projects/okutrade-hemi/index.js

```diff
@@ -0,0 +1,3 @@
+const { uniTvlExport } = require('../helper/unknownTokens')
+
+module.exports = uniTvlExport('hemi', '0xcb2436774C3e191c85056d248EF4260ce5f27A9D') //blockchain, factory address
```

Comment:
this helper is for uni V2 factories, please use `const { uniV3Export } = require("../helper/uniswapV3")`